Examples/BookingSystem.AspNetCore/Stores/FacilityStore.cs

```diff
@@ -304,6 +308,79 @@ protected override async Task GetOrderItems(List<OrderItemContext<FacilityOpport
+                            Id = RenderOpportunityId(new FacilityOpportunity
+                            {
+                                OpportunityType = OpportunityType.FacilityUse,
+                                FacilityUseId = slot.FacilityUseId
+                            }),
+                            Name = facility.Name,
+                            Url = new Uri("https://example.com/events/" + slot.FacilityUseId),
+                            Location = new Place
+                            {
+                                Name = "Fake fitness studio",
+                                Geo = new GeoCoordinates
+                                {
+                                    Latitude = facility.LocationLat,
+                                    Longitude = facility.LocationLng,
+                                }
+                            },
+                            FacilityType = new List<Concept> {
+                                    new Concept
+                                    {
+                                        Id = new Uri("https://openactive.io/facility-types#a1f82b7a-1258-4d9a-8dc5-bfc2ae961651"),
+                                        PrefLabel = "Squash Court",
+                                        InScheme = new Uri("https://openactive.io/facility-types")
+                                    }
+                                }
+                        }
+                    };
+                }
+                else
+                {
+                    slotParent = new FacilityUse
+                    {
+                        Id = RenderOpportunityId(new FacilityOpportunity
+                        {
+                            OpportunityType = OpportunityType.FacilityUse,
+                            FacilityUseId = slot.FacilityUseId
+                        }),
+                        Name = facility.Name,
+                        Url = new Uri("https://example.com/events/" + slot.FacilityUseId),
+                        Location = new Place
+                        {
+                            Name = "Fake fitness studio",
+                            Geo = new GeoCoordinates
+                            {
+                                Latitude = facility.LocationLat,
+                                Longitude = facility.LocationLng,
+                            }
+                        },
+                        FacilityType = new List<Concept> {
+                                    new Concept
+                                    {
+                                        Id = new Uri("https://openactive.io/facility-types#a1f82b7a-1258-4d9a-8dc5-bfc2ae961651"),
+                                        PrefLabel = "Squash Court",
+                                        InScheme = new Uri("https://openactive.io/facility-types")
+                                    }
+                                }
+                    };
+                }
```

Comment:
The `FacilityUse` definition seems to be duplicated here? Should we define it as a var above the `if` and then just include it in either case? More maintainable that way